hack/install.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+# Copyright 2021 VMware Tanzu Community Edition contributors. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# Inspired by - https://github.com/vmware-tanzu/community-edition/blob/main/hack/install.sh
+# Script to install tanzu framework
+# Usage: ./hack/install.sh /path/to/tanzu-framework-binary v0.10.0
```

Comment:
`/path/to/tanzu-framework-binary` is actually path to the extracted released artifacts tarball and `$VERSION` is only used in this script to form the path, since this is an intermediate state (we want to get proper install scripts part of the release artifacts), I think we can either
- update the script to expect the exact path to the binary (and not requiring version arg)
- or update the usage

Reply:
same for Windows install script

Reply:
sure, I'll update the script with the first option. This will eventually get dropped once we have the script as a release artifact.